docs/sdk/python/crypto/api-documentation.md

```diff
@@ -812,6 +812,25 @@ Get the type group of the Transaction.
 def add_payment(self, amount, recipient_id):
 ```
 
+Add a payment to the Payments array of a Transaction. Deprecated - use new add_transfer() method.
```

Comment:
`add_payment` was removed altogether.

Reply:
Thanks for review. Resolved in [ca0e45a](https://github.com/Solar-network/docs/pull/12/commits/ca0e45a01a79fd606008a7153e7e165598435120)